src/client/src/setupProxy.js

```diff
@@ -1,9 +1,9 @@
 const proxy = require('http-proxy-middleware');
 
-
 module.exports = function(app) {
     app.use(proxy('/api/**', {
-            target: process.env.IS_LOCAL === 'true' ? 'http://localhost:3333' : 'http://server:5000'
+            target: process.env.IS_LOCAL === 'true' ? 'http://localhost:5000' : 'http://server:5000',
```

Comment:
this does not work for me. trying to run local on 5000 is already taken by the docker running the server. why did we change this? @dkelley